docs/building-a-form/common-definitions.md

```diff
@@ -61,83 +66,119 @@ schema: {
         'New York',
         'Chicago'
       ]
+      }
```

Comment:
I think this one last } should be deleted!

Reply:
It didn't look right but I couldn't figure out how to check it. Thanks!